roxctl/central/debug/resync_check.go

```diff
@@ -0,0 +1,180 @@
+package debug
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path"
+	"sort"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/util"
+	"google.golang.org/grpc"
+)
+
+const (
+	resyncCheckTimeout = 300 * time.Second
```

Comment:
[nit] I would rename this to `defaultResyncCheckTimeout`

Reply:
Done